NSSNamespace/Initial_Bangazon_Site
# css changed, footer now behaves properly

## Changes:
**site.css:** gave ```.container``` class relative positioning, and ```footer``` absolute positioning and a bottom offset.  This should ensure the footer follows page content, regardless of content size or browser size.

## To test:
```git fetch```
```git checkout origin/ewFooter```

Make sure footer properly is rendered below the page content on all page views.
